tests/rules/test_set_facts.yml

```diff
@@ -14,16 +14,17 @@
             msg: hello world
             alpha: 1
             beta: 
-               name: R2D2
-               location: Naboo
+              name: R2D2
+              location: Naboo
     - name: r2
       condition: event.msg == "hello world"
       action:
         run_playbook:
           name: playbooks/validate_args_playbook.yml
-          rulebook_extra_vars:
-             fact.msg: hello world
-             fact.alpha: 1
-             fact.beta.name: R2D2
-             fact.beta.location: Naboo
+          extra_vars:
+            rulebook_extra_vars:
```

Comment:
@bzwei can we change "rulebook_extra_vars" to "my_vars" will remove confusion from earlier use of rulebook_extra_vars